internal/npm_install/npm_install.bzl

```diff
@@ -89,6 +89,10 @@ which are installed, but failures on installation are ignored.
     "exclude_packages": attr.string_list(
         doc = """DEPRECATED. This attribute is no longer used.""",
     ),
+    "ignore_peer_dependencies": attr.string_list(
+        default = [],
+        doc = """A list of peer depedencies that can be ignored.""",
```

Comment:
Could you add to this doc that `strict_peer_dependencies ` must be set for you to use this?  
Or maybe change the logic a little bit so that you can set this without needing `strict_peer_dependencies `

Reply:
How would you suggest a change the logic which can have the following 3 use cases:

`a = strict_peer_dependencies `
`b = ignore_peer_dependencies `

1. You want to ignore peer dependencies. (`a = False`, `b = []`)
1. You want to have strict dependencies and ignore a few of them. (`a = True`, `b = ['dep1', 'dep2']`)
1. You want to have strict dependencies and ignore none of them.  (`a = True`, `b = []`)

Reply:
Can you find any other popular tooling that gives a strict mode for peer deps? I would like to follow some precedent rather than invent something new.

If there is no precedent at all, maybe we shouldn't even have a strict mode - that should be someone else's job, orthogonal to whether the package manager is run by bazel

Reply:
I don't really know of any, Yarn and NPM both give warning but nothing more.

Reply:
Yeah tha's a good point @alexeagle I don't know of any that have strict deps.
Maybe we should just remove the checks, if anything it should be yarn checking things like this right, not us?

Reply:
Ok so should I revert it all and put the “true” to “false” for required?

Reply:
I have reverted and set the peer dependencies as a non requirement altogether. Is that ok?

Reply:
Yeah sounds good to me, sorry for all the back and forward.
We appciate you taking the time here!